### DISCUSSION TITLE
Pseudo death of mc mirror process

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#16911] [Q&A] [answered]
`c:\mc.exe mirror --watch --debug D:\share\recv myminio/tafile-backup/recv >>log\%date:~0,4%-%date:~5,2%-%date:~8,2%.log`

The transfer stops suddenly during the file transfer process, but the process is still there and no error is reported

mc version: [RELEASE.2022-10-29T10-09-23Z](https://github.com/minio/mc/releases/tag/RELEASE.2022-10-29T10-09-23Z)

**Comment by vadmeste** [comment, marked as answer]
@XTUxiongda you passed --watch flag, which means you asked 'mc' to keep running waiting for new uploads in the source prefix to mirror them immediately. This is a feature and not a bug.

**Reply by XTUxiongda** [reply]
> @XTUxiongda you passed --watch flag, which means you asked 'mc' to keep running waiting for new uploads in the source prefix to mirror them immediately. This is a feature and not a bug.
The problem is that the file is not transferred

**Reply by harshavardhana** [reply]
That is mostly some network issue that can't be retried. You have any logs to share?

**Reply by XTUxiongda** [reply]
![img_v2_25d46bbe-2018-4acd-b735-96a37fa0809g](https://user-images.githubusercontent.com/38322373/228778945-72dbbbdd-2067-449b-9c4b-00a7e141cd45.jpg)
I didn't see any errors reported，But the file just won't be transferred

**Reply by XTUxiongda** [reply]
@harshavardhana  do you have any idea about it?
